api/seedreconfig/seedreconfig.go

```diff
@@ -181,3 +183,15 @@ type Proxy struct {
 	// +optional
 	NoProxy string `json:"noProxy,omitempty"`
 }
+
+type AdditionalTrustBundle struct {
```

Comment:
We need to handle it in IBI too as it will break current api and IBIO, need task for it